packages.yaml

```diff
@@ -118,6 +118,7 @@ packages:
       - py-ply@3.8
       - py-virtualenv@15.1.0
       - py-pytest@3.0.7
+      - py-macs2@2.1.1.20160309
```

Comment:
This package depends on `py-numpy`, which depends on `lapack`, so this is not the correct section of the file:
```console
$ spack spec -Il py-macs2@2.1.1.20160309 ^python@2.7.13 %gcc
Input spec
--------------------------------
     2wkiioj  py-macs2@2.1.1.20160309
     7kyzqkq      ^python@2.7.13%gcc

Normalized
--------------------------------
     e7xwslg  py-macs2@2.1.1.20160309
     skamj6w      ^py-numpy@1.6:
     fpek5h6          ^py-setuptools
     5sazijq              ^python@2.7.13%gcc
     5gd5s7m                  ^bzip2
     fvevxql                  ^ncurses
     tiolp4x                      ^pkg-config
     hl37cq7                  ^openssl
     ctlbt62                      ^zlib
     xry7gic                  ^readline
     csnjmrn                  ^sqlite

Concretized
--------------------------------
     h3rtmlw  py-macs2@2.1.1.20160309%gcc@4.8.5 arch=linux-rhel7-x86_E5v4_Mellanox 
     jzdd7nj      ^py-numpy@1.12.1%gcc@4.8.5+blas+lapack arch=linux-rhel7-x86_E5v4_Mellanox 
     tubmcu2          ^openblas@0.2.19%gcc@4.8.5~openmp+pic+shared arch=linux-rhel7-x86_E5v4_Mellanox 
[+]  j4snnqr          ^py-setuptools@34.4.1%gcc@4.8.5 arch=linux-rhel7-x86_E5v4_Mellanox 
[+]  bufdij4              ^py-appdirs@1.4.3%gcc@4.8.5 arch=linux-rhel7-x86_E5v4_Mellanox 
[+]  xizdugm                  ^python@2.7.13%gcc@4.8.5+shared~tk~ucs4 arch=linux-rhel7-x86_E5v4_Mellanox 
[+]  dzql3ku                      ^bzip2@1.0.6%gcc@4.8.5+shared arch=linux-rhel7-x86_E5v4_Mellanox 
[+]  ghzrjqt                      ^ncurses@6.0%gcc@4.8.5~symlinks arch=linux-rhel7-x86_E5v4_Mellanox 
[+]  7ckck3c                          ^pkg-config@0.29.2%gcc@4.8.5+internal_glib arch=linux-rhel7-x86_E5v4_Mellanox 
[+]  4joab2d                      ^openssl@system%gcc@4.8.5 arch=linux-rhel7-x86_E5v4_Mellanox 
[+]  dsxnhm7                      ^readline@7.0%gcc@4.8.5 arch=linux-rhel7-x86_E5v4_Mellanox 
[+]  p3xykyq                      ^sqlite@3.18.0%gcc@4.8.5 arch=linux-rhel7-x86_E5v4_Mellanox 
[+]  6dnnl47                      ^zlib@1.2.11%gcc@4.8.5+pic+shared arch=linux-rhel7-x86_E5v4_Mellanox 
[+]  mqbjupa              ^py-packaging@16.8%gcc@4.8.5 arch=linux-rhel7-x86_E5v4_Mellanox 
[+]  zo5ntgm                  ^py-pyparsing@2.2.0%gcc@4.8.5 arch=linux-rhel7-x86_E5v4_Mellanox 
[+]  ux4s4op                  ^py-six@1.10.0%gcc@4.8.5 arch=linux-rhel7-x86_E5v4_Mellanox 
```